projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts

```diff
@@ -15,7 +15,7 @@ import { AfterViewInit, Directive, ElementRef, HostBinding } from '@angular/core
     selector: '.clr-form-control[vcdResponsiveInput]',
 })
 export class ResponsiveInputDirective implements AfterViewInit {
-    @HostBinding('class.clr-row') private clrGridRow = true;
+    @HostBinding('class.clr-row') public clrGridRow = true;
```

Comment:
why is this needed?

Reply:
Adds `clr-row` to the host of the directive which is required for the responsive CSS behavior.